ngraph/frontend/paddlepaddle/src/op/slice.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
+#include "slice.hpp"
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+NamedOutputs slice (const NodeContext& node) {
+    auto data = node.get_ng_input("Input");
+    auto axes = node.get_attribute<std::vector<int32_t>>("axes");
+    auto starts = node.get_attribute<std::vector<int32_t>>("starts");
+    auto ends = node.get_attribute<std::vector<int32_t>>("ends");
+    auto parialShape = data.get_partial_shape();
+    PDPD_ASSERT(parialShape.is_static(), "slice: must use static shape.");
+    auto shape = parialShape.to_shape();
```

Comment:
You only need rank here, no need to assert that shape is static, just use partialShape.rank().get_length()

Reply:
@luo-cheng2021 please have a look at this

Reply:
fixed in   ##[https://github.com/nosovmik/openvino/commit/6c451cefc4bc84c3086f1a8f37517912bbbf0f7c](url)

Reply:
fix in 0f9058cb8edab2b1cd0b740e8717ac6582eca6a3